### ISSUE TITLE
[Feature Request] Predefined Flutter steps in CI 

### ISSUE THREAD

**Issue by itsJoKr** [leoafarias/fvm#190]
I had this issue: I'm using Bitrise, which has set of predefined flutter steps (test, analyze, build...) and it's very neat to use them. Problem is that you cannot add `fvm` prefix to them, they are using flutter (hardcoded).

Now, I've solved it this way. Before the actual workflow I do these steps:

1) Install FVM (Stable flutter is already installed on CI machine)
2) Read flutter fvm version `FVM_FLUTTER_VERSION="$(grep -o '"flutterSdkVersion": "[^"]*' .fvm/fvm_config.json | grep -o '[^"]*$')"` this will return e.g. 1.22.0
3) Replace PATH to use FVM `'/Users/vagrant/fvm/versions/$FVM_FLUTTER_VERSION/bin:$PATH'`. With this I'm setting fvms flutter to first place in path so it will be preferred over already installed flutter.  

I just wanted to check with you is this the best thing I can do in this case or is there something smarter that I could done?

**Comment by leoafarias** [comment]
@itsJoKr Would something like `alias flutter=fvm flutter` work?

**Comment by itsJoKr** [comment]
Tested it, unfortunately it doesn't work. Bitrise is a bit weird here as it resets a lot of stuff between steps.

It even resets path, and I had to use their program `envman` to accomplish what I've described above. Folks from Bitrise agreed that this is right way to change path.

Anyway, my plan is to make this step public so it will appear in "step marketplace" and it's easier to share within projects.

I'll close this as it's not real issue.
